.github/ISSUE_TEMPLATE/issue_tracker_template.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Onboarding Tracker⛴️
+about: To be created by API Backlog codeowners after a new repository got approved
+title: '[Onboarding Tracker] $repo name$'
+labels: 'API Onboarding Tracker'
+assignees: ''
+
+---
+
+<!-- API & repository Tracker Issue -->
+**API information**
+- **Name**: XXXX
+- **Mail List:** XXXX
+- **Repository**: XXXX
+- **Wiki Page**: XXXX
+- **API owner**: TBD
+- **API proposal**: https://github.com/camaraproject/APIBacklog/blob/main/documentation/XXXX.md
+- **Original Issue**: https://github.com/camaraproject/APIBacklog/issues/XXXX
+
+**Action List**
+- [ ] Get nominations of initial code owner(s) (one mandatory) (API owner)
+- [ ] Update of APIBacklog.md with new status ("Onboarding")
+- [ ] Create wiki page and mailing list (send mail to support@camaraproject.org)
+- [ ] Create repository (by Admin, using automation)
```

Comment:
One additional step needed - unavoidable as the repository is dependent on wiki and mailing list info and the wiki page points back to the (created) repository:

```suggestion
- [ ] Create repository (by Admin, using automation)
- [ ] Update API Repository wiki page with GitHub repository link
```